solariot.py

```diff
@@ -141,6 +141,55 @@
     flux_client = None
     logging.info("No InfluxDB configuration detected")
 
+# Configure PVOutput
+if hasattr(config, "pvoutput_api"):
+    class PVOutputPublisher(object):
+        def __init__(self, api_key, system_id, status_url="https://pvoutput.org/service/r2/addstatus.jsp"):
```

Comment:
We bring the URL in here as we've kinda tied the payload below to the 'r2' of the API anyway, so no point making that configurable in the config file as we'd have to probably change the internals of this as well!